api/src/wfl/module/wgs.clj

```diff
@@ -51,11 +43,53 @@
     {:wgs_coverage_interval_list
      (str hg38 "wgs_coverage_regions.hg38.interval_list")}))
 
-(defn env-inputs
-  "Genome inputs for ENVIRONMENT that do not depend on the input file."
-  [environment]
+(def ^:private known-cromwells
+  ["https://cromwell-gotc-auth.gotc-dev.broadinstitute.org"
+   "https://cromwell-gotc-auth.gotc-prod.broadinstitute.org"])
```

Comment:
There are a lot of these.

Reply:
yea…. I was intentionally try to keep them in each module to make them self-sufficient, we can move them back to a centralized place if we find more patterns between them?

Reply:
OK by me.